server/platform-support/platform-support.rst

```diff
@@ -741,3 +741,12 @@
 
 .. [#oracle-support] On Oracle Linux, MongoDB only supports the Red Hat
    Compatible Kernel.
+
+Starting in MongoDB 8.0, new MongoDB Server versions (major and minor)
+support the minimum operating system (OS) minor version defined by the
+OS vendor. After an OS minor version is no longer supported by the OS
+vendor, MongoDB updates the MongoDB Server to support the next OS minor
+version. For details, see:
+
```

Comment:
(Content)
There is nothing about this paragraph that communicates a change in version from Amazon2023 to 2023.3 (x86 and arm64). What is your reasoning for omitting that?

Reply:
Per our discussion with PM and dev, will not doc minor versions.